frontend/src/components/Main/index.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { FC } from "react";
-import "./index.scss";
 import Header from "../Header";
-
-const Main: FC = ({ children }) => {
+import Footer from "../Footer";
+export const Main: FC = ({ children }: any) => {
   return (
-    <section className="main-section">
+    <body data-testId={"mainBody"}>
```

Comment:
testId? is this debug code left in?

Reply:
data- anything is custom properties for an HTML element.  In this case I am using a custom property called testId which I use to look for in a test script.

Reply:
const { getByTestId } = render(<Main />);
    const mainBody = getByTestId("mainBody");
    expect(mainBody).toBeTruthy();

Reply:
thats the clarification i was looking for, thank you.